vehicle/docker-compose.yml

```
@@ -1,64 +1,67 @@
 x-autoware-base: &autoware-base
   image: "aichallenge-2025-dev-${USER}"
   privileged: true
+  pull_policy: never
   network_mode: host
   environment:
     - DISPLAY=${DISPLAY}
     - USER=${USER}
     - ROS_DISTRO=humble
+    - XAUTHORITY=${XAUTHORITY}
+    - QT_X11_NO_MITSHM=1
+    - TZ=Asia/Tokyo
   volumes:
     - ../output:/output
     - ../aichallenge:/aichallenge
     - ../remote:/remote
     - ./:/vehicle
     - /tmp/.X11-unix:/tmp/.X11-unix:rw
     - /dev/dri:/dev/dri
+    - ${XAUTHORITY}:${XAUTHORITY}:rw
   devices:
     - /dev/dri
+    - /dev/video0
   working_dir: /aichallenge
 
+x-racing_kart_interface-base: &racing_kart_interface-base
+  image: "ghcr.io/tier4/racing_kart_interface:latest-experiment"
```

まあ、このままでもいいのかもしれない。。。
```suggestion
  image: "ghcr.io/tier4/racing_kart_interface:latest-experiment"
```